frontend/app/pages.js

```diff
@@ -90,6 +93,15 @@ export function waitingGame(gameID) {
             }
         }
     });
+
+    // Listen for a gameStart event coming from the server.
+    sseEventSource.addEventListener('gameStart', (startEvent) => {
+        const startedGameId = JSON.parse(startEvent.data);
```

Comment:
For this line, is it necessary to have JSON.parse()? Because the gameStart event's data is simply a **string** like the following: "data: X" (where X is the game id).

If it works as it is, then there's no need to change it, I'm just being picky.

Reply:
Your right, it doesn't need it. Thanks for picking that out.